src/client/Microsoft.Identity.Client/Internal/Broker/BrokerParameter.cs

```diff
@@ -36,6 +36,7 @@ internal class BrokerParameter
 
         public const string BrokerInstallUrl = "broker_install_url";
         public const string BrokerV2 = "msauthv2://";
+        public const string BrokerV3 = "msauthv3://";
```

Comment:
These constants are uri schemes, so they should be named appropriately.

Reply:
good point. thanks,

---
In reply to: [323181834](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1383#discussion_r323181834) [](ancestors = 323181834)